src/test/java/org/opensearch/security/SnapshotRestoreTests.java

```diff
@@ -231,106 +240,145 @@ public void testSnapshotCheckWritePrivileges() throws Exception {
         Assert.assertEquals(HttpStatus.SC_OK, rh.executePostRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"/_restore?wait_for_completion=true&pretty","{ \"rename_pattern\": \"(.+)\", \"rename_replacement\": \"restored_index_$1\" }", encodeBasicHeader("snapresuser", "nagilum")).getStatusCode());
-        
+
         putSnapshot =
-        "{"+
-          "\"indices\": \".opendistro_security\","+
-          "\"ignore_unavailable\": false,"+
-          "\"include_global_state\": false"+
-        "}";
-                
-        Assert.assertEquals(HttpStatus.SC_OK, rh.executePutRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"?wait_for_completion=true&pretty", putSnapshot, encodeBasicHeader("snapresuser", "nagilum")).getStatusCode()); 
+                "{"+
+                        "\"indices\": \".opendistro_security\","+
+                        "\"ignore_unavailable\": false,"+
+                        "\"include_global_state\": false"+
+                        "}";
+
+        Assert.assertEquals(HttpStatus.SC_OK, rh.executePutRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"?wait_for_completion=true&pretty", putSnapshot, encodeBasicHeader("snapresuser", "nagilum")).getStatusCode());
         Assert.assertEquals(HttpStatus.SC_FORBIDDEN, rh.executePostRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"/_restore?wait_for_completion=true&pretty","{ \"rename_pattern\": \"(.+)\", \"rename_replacement\": \"restored_index_$1\" }", encodeBasicHeader("snapresuser", "nagilum")).getStatusCode());
-              
+
+
         putSnapshot =
-        "{"+
-          "\"indices\": \"testsnap2\","+
-          "\"ignore_unavailable\": false,"+
-          "\"include_global_state\": true"+
-        "}";
-                        
-        Assert.assertEquals(HttpStatus.SC_OK, rh.executePutRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"?wait_for_completion=true&pretty", putSnapshot, encodeBasicHeader("snapresuser", "nagilum")).getStatusCode()); 
+                "{"+
+                        "\"indices\": \"testsnap2\","+
+                        "\"ignore_unavailable\": false,"+
+                        "\"include_global_state\": true"+
+                        "}";
+
+        Assert.assertEquals(HttpStatus.SC_OK, rh.executePutRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"?wait_for_completion=true&pretty", putSnapshot, encodeBasicHeader("snapresuser", "nagilum")).getStatusCode());
         Assert.assertEquals(HttpStatus.SC_FORBIDDEN, rh.executePostRequest("_snapshot/bckrepo/"+putSnapshot.hashCode()+"/_restore?wait_for_completion=true&pretty","{ \"include_global_state\": true, \"rename_pattern\": \"(.+)\", \"rename_replacement\": \"restored_index_$1\" }", encodeBasicHeader("snapresuser", "nagilum")).getStatusCode());
     }
 
+    @Test
```

Comment:
Nice job writing a test for this in the security repo! This looks good to me. If we are to add more scenarios for testing than I would suggest breaking these down into individual tests.

Reply:
That makes sense. I will break it out if we add more. The current test framework is a little weird with three distinct formats depending on the era it was written.